test/extended/cluster/cl.go

```diff
@@ -74,11 +74,40 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 				e2e.Logf("Our tuning set is: %v", tuning)
 			}
 			for j := 0; j < p.Number; j++ {
-				// Create namespaces as defined in Cluster Loader config
-				nsName := fmt.Sprintf("%s%d", p.Basename, j)
-				err := oc.Run("new-project").Args(nsName).Execute()
+				nsName := p.Basename
+				if len(p.IfExists) == 0 {
+					e2e.Logf("Parameter 'ifexists' not specified. Using 'default' value.")
+					p.IfExists = IF_EXISTS_DEFAULT
```

Comment:
Don't really need to have an alternate value that is functionally equal to nil.

Reply:
Done.